httpserver/src/main/kotlin/tbdex/sdk/httpserver/handlers/GetExchange.kt

```diff
@@ -88,7 +87,7 @@ suspend fun getExchange(
   if (callback != null) {
     // TODO: figure out what to do with callback result. should we pass through the exchanges we've fetched
     //       and allow the callback to modify what's returned? (issue #10)
-    val result = callback.invoke(call, GetExchangesFilter())
```

Comment:
i don't think we need to pass in a filter for get single exchange since that's only asking for 1 exchange